CHANGELOG.md

```diff
@@ -0,0 +1,169 @@
+# Changelog
+
+> Specification changelog.
+
+## v2022.12
+
+### Updates
+
+#### APIs
+
```

Comment:
This would be a lot easier to read if there were a single bullet point for "add complex number support to existing APIs", and then indented bullets per API like "`__abs__`: ([gh-546](https://github.com/data-apis/array-api/pull/546))". That way the other changes are easy to find, how they're too hidden.